### ISSUE TITLE
Storybook compatibility and simpler npm integration

### ISSUE THREAD

**Issue by msand** [ipselon/structor-market#3]
I was thinking, it would be good to add support/compatibility with storybook, such that when making a spkg of a npm library, it could take all the components and examples from the pre-existing structure. What do you think? 

Making a spkg from a npm package, which uses storybook to showcase/test their components, could be reduced to adding one object to structor-market/index.json.

**Comment by msand** [comment]
Actually, it might be possible to reduce the npm integration to adding a "spkg" keyword to the keywords array in package.json https://docs.npmjs.com/files/package.json#keywords
This would allow searching for structor packages using npm search.

**Comment by ipselon** [comment]
Hmm... thinking what capabilities Structor has, we may see that it's doing the same things with Storybook. It has hot loading, developer can see own components in isolation. 

What the goal should be achieved by integrating with Storybook?

**Comment by msand** [comment]
Well, the idea is that several component libraries having examples in the storybook format already exist. If Structor can interpret that, then the can be used for prototyping in Structor models as is, without having to create a component library with examples separately in the spkg format. This would expand the number of available components / examples which can be used significantly.

**Comment by ipselon** [comment]
Good point. What do you think if Structor will be able to create and export a presentation of pages which can be uploaded to some hosting?
